level-packager.js

```diff
@@ -15,8 +15,8 @@ function packager (leveldown) {
 
   [ 'destroy', 'repair' ].forEach(function (m) {
     if (typeof leveldown[m] === 'function') {
-      Level[m] = function (location, callback) {
-        leveldown[m](location, callback || function () {})
+      Level[m] = function (location, env, callback) {
+        leveldown[m](location, env || 0, callback || function () {})
```

Comment:
should it not be `undefined` instead of `0` ?
What is the `0` used for, in the leveldown wrapper ?

Reply:
this number is acting as a pointer, so i guess its for a null pointer, not sure yet, need to go to leveldown for that ^^

Reply:
@alexandre-merle @DavidPineauScality: You're right, if the environment is a NULL pointer, the default environment is used in place in LevelDown.